plugins/source/postgresql/resources/plugin/plugin_test.go

```diff
@@ -103,16 +107,16 @@ func getTestCases(serialValue int64) []testCase {
 	if err != nil {
 		panic(err)
 	}
-	timestampMillisecond := scalar.Timestamp{
-		Type: &arrow.TimestampType{
-			Unit:     arrow.Millisecond,
-			TimeZone: "UTC",
-		},
-	}
-	err = timestampMillisecond.Set("1999-01-08 04:05:06.789")
-	if err != nil {
-		panic(err)
-	}
+	// timestampMillisecond := scalar.Timestamp{
```

Comment:
Needs https://github.com/cloudquery/plugin-sdk/pull/986

Reply:
Can be re-added now?

Reply:
Good catch, done in https://github.com/cloudquery/cloudquery/pull/11719/commits/482c8f264089fcd8931fb8cdf0f3d8a50c645eb7